src/ProfileEndpoints.jl

```diff
@@ -1,6 +1,7 @@
 module ProfileEndpoints
 
 import HTTP
+import JSON
```

Comment:
I _think_ we should always be using `JSON3` instead of `JSON` these days.

RAICode should have JSON3 in it already anyway as well.